tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/SDK/RuntimePipelineInvokeAsyncIntegration.cs

```diff
@@ -50,6 +52,15 @@ public class RuntimePipelineInvokeAsyncIntegration
                 tags.Region = executionContext.RequestContext.ClientConfig.RegionEndpoint?.SystemName;
             }
 
+            if (scope != null && Serverless.IsRunningInLambda() && LambdaCommon.IsSyncLambdaInvocation(executionContext.Instance))
+            {
+                LambdaCommon.InjectTraceContext(
+                    executionContext.Instance,
+                    scope.Span.TraceId.ToString(),
+                    scope.Span.SpanId.ToString(),
+                    Tracer.Instance.TracerManager.Sampler?.GetSamplingPriority(scope.Span));
```

Comment:
Is this the proper way to get sampling priority from the tracer? I tried checking `scope.Span.Context.SamplingPriority`, but it didn't have a value

Reply:
Your first hunch was correct, but that value can be null if unset. I'll post a related comment at `LamdbaCommon.InjectTraceContext` to offer a suggestion on how you could rework this.